js/everything.js

```diff
@@ -651,6 +651,11 @@ $(function() {
         $("#demo").hide();
     }
 
+    // Reverts to Home when there is no text in input
+    if ($(".tt-menu").hasClass("tt-empty")) {
```

Comment:
This logic is correct in theory, but we need to do the check inside an event handler when the text in `.tt-menu` changes. I'll let you figure this one out, it might end up looking something like:

```js
$(".tt-menu").on("input", function(e) {
    // Move your logic here
});
```

You should definitely be able to test locally, read `CONTRIBUTING.md` for setup steps.